web/src/lib/components/faces-page/show-hide.svelte

```diff
@@ -1,24 +1,46 @@
+<script lang="ts" context="module">
+  export enum ToggleVisibilty {
+    HIDE_ALL = 'hide-all',
+    HIDE_UNNANEMD = 'hide-unnamed',
+    VIEW_ALL = 'view-all',
+  }
+
+  export const changeVisibility = (toggleVisibility: ToggleVisibilty) => {
```

Comment:
This method and the whole data flow because of this is kinda confusing to me. I think I've got an idea to make it more intuitive/simpler:

- `onChange` passes the next visibility option (and the old one if necessary), so `onChange: (next: ToggleVisibility) => void;`
- This `changeVisibility` will be moved "inside" the component and not get exported
- `onChange`will be called with something like `getNextVisibility(toggleVisibility)`

Reply:
That's better, thanks !